life_expectancy/utils/load_save_data.py

```diff
@@ -0,0 +1,66 @@
+""" Load and save data script """
+#--------------------------------------------------------------------
+# IMPORTS
+#--------------------------------------------------------------------
+from abc import ABC, abstractmethod
+from zipfile import ZipFile
+import json
+import pandas as pd
+
+#--------------------------------------------------------------------
+# ABSTRACT CLASS (LOADING)
+#--------------------------------------------------------------------
+
+class DataLoaderStrategy(ABC):
+    """Abstract class for loading data"""
+    @abstractmethod
+    def load_data(self, file_path):
+        """Abstract method to load the data"""
+
+#--------------------------------------------------------------------
+# CLASS FOR TSV (LOADING)
+#--------------------------------------------------------------------
+
+class TSVLoader(DataLoaderStrategy):
+    """class for loading TSV data"""
+
+    def load_data(self, file_path) -> pd.DataFrame:
+        '''
+        Loads the csv file that contains the data
+        returns:
+            data(Pandas DataFrame): Loaded dataframe
+        '''
```

Comment:
You are not consistent in the docstrings. Sometimes you use `'`, but others `"`. That's okay since the formatter is not checking for that 😁